views/models/ChartModel/index.js

```diff
@@ -1,5 +1,15 @@
+import { SAFE_URL_PATTERN, DATA_URL_PATTERN } from '../../../constants'
+
 const ChartModel = {
   isOwnedByUser: (chart, user) => chart.owner === user.uid,
+  sanitiazeProfileImageUrl: (url = 'about:blank') => {
```

Comment:
do we need to default this?

Reply:
The default was added in the example code I gave, but that was more generic so I think here it's unnecessary.

Reply:
[05e9f09](https://github.com/AMP-SCZ/dpdash/pull/530/commits/05e9f090fc738b3090970d1e415c0b1966628835)